docs/source/create_channel/create_channel_genesis.md

```diff
@@ -0,0 +1,283 @@
+
+Policies also govern the **processes** of how users interact with the channel, such as the set of organizations that need to approve a chaincode before it can be deployed to a channel, which channel members can invoke a chaincode to write data to the channel, and which actions need to be completed by channel administrators.
+
+Channel policies are important enough that they need to be discussed in their own topic. Unlike other parts of the channel configuration, the policies that govern the channel are determined by by how different sections of the `configtx.yaml` file work together. While channel policies can be configured for any use case with few constraints, this topic will focus on how to use the default policies provided by Hyperledger Fabric. By default, Fabric uses a combination of signature policies, ImplicitMeta policies, and Access Control Lists to determine how channel members participate in the channel processes and update the channel structure. You can learn more about the role of policies in Hyperledger Fabric by visiting the [Policies concept topic](../policies.html).
+
+Each channel member is associated with a set of signature policies. Signature policies read the the digital signatures that are attached to transactions or channel update requests that are submitted to the channel. Each signature policy lists a set of organizations and roles that will satisfy if those signatures are found in the transaction. You can see the set signature policies associated with peer organization Org1 in the **Organizations** section of `configtx.yaml` below:
+```yaml
+- &Org1
+
+  ...
+
+  Policies:
+      Readers:
+          Type: Signature
+          Rule: "OR('Org1MSP.admin', 'Org1MSP.peer', 'Org1MSP.client')"
+      Writers:
+          Type: Signature
+          Rule: "OR('Org1MSP.admin', 'Org1MSP.client')"
+      Admins:
+          Type: Signature
+          Rule: "OR('Org1MSP.admin')"
+      Endorsement:
+          Type: Signature
+          Rule: "OR('Org1MSP.peer')"
+```
+
+Org1 is associated with four sets of signature policies. The policies can only be satisfied by signatures from Org1. However, each policy lists a different set of roles from within the organization that satisfy the policy. The `Admins` policy can only be satisfied by the transactions submitted by an admin identity, while only peers can satisfy the `Endorsement` policy. Each transaction can satisfy multiple policies. If a smart contract was invoked and endorsed by a peer from Org1 and Org2, the transaction would satisfy the `Endorsement:` policy that was associated with both organizations.
```

Comment:
I need more information for this point `If a smart contract was invoked and endorsed by a peer from Org1 and Org2, the transaction would satisfy the `Endorsement:` policy that was associated with both organizations.` I'm not following based on the example above.

Reply:
Is this clearer: "Each transaction can satisfy multiple policies. For example, if a transaction was endorsed by a peer from Org1 and Org2, the transaction would satisfy the `Endorsement:` signature policy associated with both organizations."